README.md

```diff
@@ -17,20 +17,20 @@ If you're having trouble downloading & installing the bridge, see the [Known Iss
 ### Importing your Xcode Project
 When using the bridge, the first thing you'll want to do is generate a Visual Studio solution from your Xcode project:
 
-1. Open the Windows Command Prompt (type **cmd** in the Start Menu to find it) and navigate to your Xcode project directory using the **cd** command. Note that you should go to the directory containing your project *.xcworkspace* or *.xcodeproj* folder: you shouldn't be inside those folders, but one level above them.
+1. Open the Windows Command Prompt (type **cmd** in the Start Menu to find it) and navigate to your Xcode project directory using the **cd** command. Note: use the directory containing your .workspace or .xcodeproj folder.
 
   ```
-    cd C:\winobjc\samples\MyProject
+    [location of unzipped SDK]\MyProject> cd [location of unzipped SDK]\winobjc\samples\MyProject
```

Comment:
Wrong location for prompt. This suggests you start in a non-existent directory in the SDK.

Suggest:
**c:\> cd c:\myObjCProject**